src/main/kotlin/net/maizegenetics/phgv2/pathing/HaplotypeEmissionProbability.kt

```diff
@@ -63,7 +62,8 @@ class HaplotypeEmissionProbability(val refRangeToHapListMap : Map<ReferenceRange
         if (myHapMappings.isNullOrEmpty()) {
             //if there no reads then all haplotypes get assigned the same probability, it does not matter what it is.
             //note that 0.0 is ln(1), so this actually assigns a probability of 1 not zero. One interpretation is that
-            //if there are no reads then every haplotype will have 0 reads with a probability of 1.
+            //if there are no reads then every haplotype (including null haplotypes) will have 0 reads with a probability of 1.

```

Comment:
Comment says probability of 1, but variable is 0.0?

Reply:
The comment line before that points out that 0.0 = ln(1).